mappings/net/minecraft/world/World.mapping

```diff
@@ -62,6 +77,7 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 4 sizeX
 		ARG 5 sizeY
 		ARG 6 sizeZ
+	METHOD method_218 isWater (Lnet/minecraft/class_25;)Z
```

Comment:
to align with "isBoxSubmergedInFluid" should be "isBoxSubmergedInWater"
especially since it gives an indication that the box is entirely water.